couchbase/operations/management/collections_manifest_get.hxx

```diff
@@ -37,6 +37,7 @@ struct collections_manifest_get_response {
 struct collections_manifest_get_request {
```

Comment:
Should this be under management, it's an mcbp operation

Reply:
yes, because the nature of the operation is rather related to management area.